### PR TITLE
Add instructions for running TDR reporting

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: manual/tdr-reporting.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Run the reports
+Go to the [tdr-reporting actions page](https://github.com/nationalarchives/tdr-reporting/actions/workflows/run.yml). You will see a page similar to this
+
+![](./images/reporting_run_action.png)
+ 
+Click "Run workflow" which will give you options like this:
+
+![](./images/reporting_drop_down.png)
+You can ignore the first dropdown with Branch: main in it. The other three can be configured:
```
Comment (TomJKing) [inline manual/tdr-reporting.md:20]: Think a line break is needed after the image link. When I view the page, the test is appearing on the same line as the bottom of the image.

File: manual/tdr-reporting.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### Report Type
+#### Standard
+This is a detailed report containing all TDR consignments and includes the transferring body code, export date time and a file count.
+
+#### CaseLaw
+This report has fewer details than the standard report and only shows judgment transfers.
+
+### Emails
+The job will send the report through to Slack so this needs to be the email address associated with your TNA slack account. You can add more than one email address for multiple recipients but each needs to be enclosed in double quotes and separated by a comma. For example:
+"user1@nationalarchives.gov.uk"
```
Comment (TomJKing) [inline manual/tdr-reporting.md:42]: Might want to put the two different examples as list items, as it looks a bit weird on the page as it is:
`* "user1@nationalarchives.gov.uk"`;
`* "user1@nationalarchives.gov.uk","user2@nationalarchives.gov.uk"`

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor formatting issues.

File: manual/tdr-reporting.md (hunk 1 above)
Comment (MancunianSam) [reply manual/tdr-reporting.md:20]: I've put the line break in, cheers.

File: manual/tdr-reporting.md (hunk 2 above)
Comment (MancunianSam) [reply manual/tdr-reporting.md:42]: Makes sense, I've changed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**